R/method-fortify.R

```diff
@@ -394,11 +394,15 @@ fortify.phylo4 <- function(model, data, layout="rectangular", yscale="none",
 ##' @method fortify phylo4d
 ##' @export
 fortify.phylo4d <- function(model, data, layout="rectangular", yscale="none",
-                            ladderize=TRUE, right=FALSE, mrsd=NULL, ...) {
+                            ladderize=TRUE, right=FALSE, branch.length="branch.length",
```

Comment:
The `branch.length` parameter may be passed by `...`, but it is ignored later on in the chain. The change I implemented allows ggtree to scale branch lengths of phylo4d objects.